pkgpanda/__init__.py

```diff
@@ -356,7 +356,7 @@ def load(self, id: str):
         filename = os.path.join(path, "pkginfo.json")
         try:
             pkginfo = load_json(filename)
-        except FileNotFoundError as ex:
+        except OSError as ex:
```

Comment:
1. We aren't looking for a specific error here. We're trying to raise an exception with a meaningful message to aid debugging in the event that a `pkginfo.json` can't be read, for whatever reason. This is why we include the exception's error message in the `PackageError` that we raise -- we aren't certain what the cause is until runtime.
1. This would make the `PackageNotFound` exception that gets raised ambiguous. If we catch the error here, we get a more detailed error with a clearer cause.